src/pretix/control/templates/pretixcontrol/checkin/index.html

```diff
@@ -180,13 +200,16 @@ <h3 class="panel-title">{% trans "Filter" %}</h3>
             </div>
             {% if "can_change_orders" in request.eventpermset %}
                 <button type="submit" class="btn btn-primary btn-save">
+                    <span class="fa fa-sign-in" aria-hidden="true"></span>
                     {% trans "Check-In selected attendees" %}
                 </button>
                 <button type="submit" class="btn btn-default btn-save" name="checkout" value="true">
+                    <span class="fa fa-sign-out" aria-hidden="true"></span>
                     {% trans "Check-Out selected attendees" %}
                 </button>
                 <button type="submit" class="btn btn-default btn-save" name="revert" value="true">
```

Comment:
Not sure this is needed, but to make it consistent with e.g. bulk-deleting of events, this should be a red button.

```suggestion
                <button type="submit" class="btn btn-danger btn-save" name="revert" value="true">
```